sdk/ml/azure-ai-ml/tests/test_configs/deployments/batch/batch_component_deployment.yml

```diff
@@ -0,0 +1,12 @@
+$schema: http://azureml/sdk-2-0/BatchDeployment.json
+name: component-deployment
+description: A sample deployment with pre and post processing done before and after inference.
+endpoint_name: component-endpoint
+compute: azureml:cpu-cluster
+type: component
+job_definition:
+    type: pipeline
+    component: ../../components/component_e2e.yml
+    # component: azureml:atestcomponent:1
```

Comment:
It is worth adding another test yaml with this short form!

Reply:
Will add once backend changes are ready